packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/TextList.test.js

```diff
@@ -13,28 +13,34 @@
 // limitations under the License.
 
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
 import TextList from './TextList';
 
 describe('<TextList>', () => {
-  let wrapper;
-
-  const data = [
-    { key: 'span.kind', value: 'client' },
-    { key: 'omg', value: 'mos-def' },
-  ];
+  const data = ['string 1', 'string 2', 'another string'];
```

Comment:
This is what we get for not having tests in Typescript :-(
The old shape didn't make sense for the component.